risc0/zkvm/src/exec/mod.rs

```diff
@@ -129,6 +130,11 @@ impl Write for Journal {
 
 impl<'a> Executor<'a> {
     /// Construct a new [Executor] from a [MemoryImage] and entry point.
+    /// Before a guest program is proven, the [Executor] is responsible for
+    /// deciding where a zkVM program should be split into [Segment]s and what
+    /// work will be done in each segment. This is the execution phase:
+    /// the guest program is executed to determine how its proof should be
+    /// divided into subparts.
     pub fn new(env: ExecutorEnv<'a>, image: MemoryImage, pc: u32) -> Self {
```

Comment:
Now that the PC is part of the MemoryImage, we should drop the `pc` argument here